ema_workbench/analysis/parcoords.py

```diff
@@ -143,6 +143,27 @@ class ParallelAxes:
     rot : float, optional
           rotation of axis labels
 
+
+    Attributes
+    ----------
+    limits : DataFrame
+    recoding : dict
+               non numeric columns are converting to integer variables
+    flipped_axes : set
+                   set of Axes that are to be shown flipped
+    axis_labels : list of str
+    fontsize : int
+    normalizer : MinMaxScaler instance
+    fig : a matplotlib Figure instance
+    axes : list of matplotlib Axes instances
+    ticklabels : list of str
+    datalabels : list of str
+                 labels associated with lines
+
+
+    The basic setup of the Parallel Axis plot is a row of mpl Axes instances, with all whitespace
```

Comment:
Should this line have a header, since from here class methods are discussed?

Reply:
moved it into Notes which is the proper place for implementation details